gradio/components/dataframe.py

```diff
@@ -17,11 +17,9 @@
 
 
 class DataframeData(GradioModel):
-    headers: List[str]
+    headers: Optional[List[str]] = None
```

Comment:
oo Interesting when did we make the headers optional?

Reply:
actually you're right they're not, will revert this part.